### ISSUE TITLE
Screen Reader does not read step titles

### ISSUE THREAD

**Issue by aaronprisk** [canonical/ubuntu-desktop-provision#801]
The screen reader that can be enabled at the start of the installer does not read the title and subtitle of each installer step. 

Example: "Select your keyboard layout" "How would you like to install Ubuntu?"

![Screenshot_20240823_193609](https://github.com/user-attachments/assets/caef2707-37de-4f67-ad23-7223d8ee197f)

**Comment by Joel8601** [comment]
any developments?
any ways to edit the source code and make it possible?

**Comment by sminez** [comment]
We recently had an accessibility audit carried out and are working through the findings over the coming months. Things like screen-reader support are in there but will need to be addressed on a case by case basis I think.

**Comment by archisman-panigrahi** [comment]
> will need to be addressed on a case by case basis

Can I turn on the screen reader, close my eyes, and try to install ubuntu, and give a feedback about what seems to be lacking?

**Comment by Joel8601** [comment]
I don't know how to explain this in words, but whenever we try to install Ubuntu-mate with screen-reader (Orca)
when the installer wizard opens up the screen says "Ubuntu-mate installer frame " and after we hit tab the focus directly shifts to the any button present and the screen reader just reads out push button and  nothing else a blind person cannot identify what this push is for. Lets say for example the image above is a page in installer to whether to select to install any third party apps along with Ubuntu-mate but the screen reader only reads out the two options and not the highlighted text, similar problems exist in other pages too but they all are not common except some, a case by case approach would be nice.but how soon?

**Comment by archisman-panigrahi** [comment]
Another thought: Please check the accessibility features offered by the Windows installer, and ensure that the Ubuntu installer provides similar features.

**Comment by sergio-costas** [comment]
I can add to this that the list of options in the "Accessibility" tab only says "Pushbutton", instead of "Seeing pushbutton", "Hearing pushbutton"...

**Comment by paulGeoghegan** [comment]
Hi there, can confirm. What happens is the user just hears header read to them at least in 25.10 but that's it. This is consistent across every page so I think this issue should be higher on the list in terms of priority.
